assets/xml/scenes/overworld/spot08.xml

```diff
@@ -1,6 +1,9 @@
 <Root>
     <File Name="spot08_scene" Segment="2">
         <Cutscene Name="gZorasFountainIntroCs" Offset="0x4A80"/>
+        <Cutscene Name="gZorasFountainSapphireCs" Offset="0x2D10"/>
+        <Cutscene Name="gZorasFountainUnusedJabuCs" Offset="0x3C80"/>
+        <Cutscene Name="gZorasFountainCreditsCs" Offset="0x41F0"/> <!-- Note: unused in the credits sequence -->
         <Scene Name="spot08_scene" Offset="0x0"/>
```

Comment:
should we just put unused in the name? we are already doing it in other places

Reply:
I thought of it and I remembered once I was told to remove it actually, because the cutscene was technically accessible from the `CS_DESTINATION` feature

that said I don't really care, I'll make the change if that's ok

Reply:
Oh I see, yeah if its referenced from code I guess "unused" is a bit ambiguous. Im fine with a comment

Reply:
Are you referring to https://github.com/zeldaret/oot/pull/2338#discussion_r1879348291 ? That one is just straight up not an unused cutscene. It plays immediately after the first time you pull out the Master Sword and is how you normally branch into the Light Medallion cutscene.

`CS_DEST_ZORAS_FOUNTAIN_CREDITS` has zero hits outside of z_demo.c, it is ___actually___ unreachable and thus the cutscene should be documented as being unused.

Reply:
oh I thought the temple of time version was the unused one, my bad then